lib/default-theme/NavLink.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <router-link
+    class="router-link"
+    :to="item.link"
+    v-if="!isExternal(item.link)">
```

Comment:
better put the `>` on the next line, stuck to `{{ item.text }}` and stick the next `router-link` to the end of it too. But I see it comes this way from the original code 🤔

Reply:
Yeah, that's pretty much what I said

Reply:
cool, fixed at 7eb2563